### DISCUSSION TITLE
No revert button in merged pull request page

### DISCUSSION THREAD

**Discussion by aminlatifi** [community/community#23693] [New to GitHub]
<p>Hi,<br>
I have admin role in repository, but I cannot revert pull request merged by another user. I don’t see revert button in this pull request page as other ones.<br>
</p><aside class="onebox githubpullrequest">
  <header class="source">
      <a href="https://github.com/Giveth/feathers-giveth/pull/234" target="_blank" rel="noopener nofollow ugc">github.com/Giveth/feathers-giveth</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Pull Request">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 12 16" aria-hidden="true"><path d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/Giveth/feathers-giveth/pull/234" target="_blank" rel="noopener nofollow ugc">Added PAN token to defaults.json, added coingeckoid property, impleme…</a>
    </h4>

    <div class="branches">
      <code>Giveth:develop</code> ← <code>arbreton:f_coinGeckoAPI</code>
    </div>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2020-09-13" data-time="04:08:12" data-timezone="UTC">04:08AM - 13 Sep 20 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/arbreton" target="_blank" rel="noopener nofollow ugc">
          <img alt="arbreton" src="https://user-images.githubusercontent.com/5684607/181078776-0e32d6c7-fd4f-4984-83e9-98d324015041.png" class="onebox-avatar-inline" width="20" height="20">
          arbreton
        </a>
      </div>

      <div class="lines" title="2 commits changed 2 files with 93 additions and 8 deletions">
        <a href="https://github.com/Giveth/feathers-giveth/pull/234/files" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+93</span>
          <span class="removed">-8</span>
        </a>
      </div>
    </div>

  </div>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by francisfuzz** [comment]
<blockquote>
<p>I have admin role in repository, but I cannot revert pull request merged by another user. I don’t see revert button in this pull request page as other ones.</p>
</blockquote>
<p>:wave: <a class="mention" href="/aminlatifi">@aminlatifi</a> , welcome to the GitHub Support Community! Since you opened this, it looks like you’ve already reverted that pull request in <a href="https://github.com/Giveth/feathers-giveth/pull/238">https://github.com/Giveth/feathers-giveth/pull/238</a>. If you’re still able to reproduce this behavior (absence of a Revert button despite having admin permissions) in a new test repository, could you share instructions for doing so? That would help us investigate and possibly get in touch with our engineering team for additional help (if needed).</p>

**Reply by agaald1** [reply]
Hi, I experience the same behaviour. I have admin rights on a repository, and there is no revert button for a merged pull request. 
Is there a way to enable it?

**Comment by aminlatifi** [comment, marked as answer]
<p>Thanks for your response,<br>
I reverted that PR manually</p>
